signup/tool/src/java/org/sakaiproject/signup/tool/jsf/SignupSiteWrapper.java

```diff
@@ -105,6 +105,7 @@ public boolean isAllowedToCreate() {
 	 *            a list of SignupGroupWrapper objects.
 	 */
 	public void setSignupGroupWrappers(List<SignupGroupWrapper> groupWrappers) {
+		groupWrappers.sort((SignupGroupWrapper x, SignupGroupWrapper y) -> x.getSignupGroup().getTitle().compareTo(y.getSignupGroup().getTitle()));

```

Comment:
Sorting should be before this method is called.

Reply:
I recommend to use a Collator instead, here is an example #5526

Reply:
thank you both, I'll do this in that way, it'll be better